cinnabar/tests/test_measurements.py

```diff
@@ -12,6 +13,30 @@ def test_ground():
     assert g1 != g3
     assert g3 == g4
 
+@pytest.mark.parametrize('Ki,uncertainty,dG,dG_uncertainty,label,temp', [
+    [100 * unit.nanomolar, 10 * unit.nanomolar, -9.55 * unit.kilocalorie_per_mole,
+     0.059 * unit.kilocalorie_per_mole, 'lig', 298.15 * unit.kelvin],
+    [0.1 * unit.micromolar, 0.01 * unit.micromolar, -9.55 * unit.kilocalorie_per_mole,
+     0.059 * unit.kilocalorie_per_mole, 'lig', 298.15 * unit.kelvin],
+    [100 * unit.nanomolar, 10 * unit.nanomolar, -10.57 * unit.kilocalorie_per_mole,
+     0.066 * unit.kilocalorie_per_mole, 'lig', 330 * unit.kelvin],
+])
+def test_Ki_to_DG(Ki, uncertainty, dG, dG_uncertainty, label, temp):
+
+    Ki_to_DG = cinnabar.Measurement.from_experiment(Ki, label, uncertainty, '', temp)
+
+    assert pytest.approx(dG, 0.001) == Ki_to_DG.DG
+    assert pytest.approx(dG_uncertainty, 0.01) == Ki_to_DG.uncertainty
+
+def test_negative_Ki():
+
+    with pytest.raises(ValueError, match=r"Ki value cannot be zero or negative. Check if dG value was provided instead of Ki."):
```

Comment:
We try (and often fail to), stick to the pep8 requirements (i.e. in this case 2 empty lines between functions and a maximum of 79 characters). Could you ammend these to match this?

Reply:
Changed this!